lib/rules/function-url-quotes/index.js

```diff
@@ -19,8 +19,22 @@ const meta = {
 	url: 'https://stylelint.io/user-guide/rules/function-url-quotes',
```

Comment:
[suggest] `fixable: true` in the metadata is required:

```suggestion
	url: 'https://stylelint.io/user-guide/rules/function-url-quotes',
	fixable: true,
```